bundlewrap/secrets.py

```diff
@@ -319,6 +322,13 @@ def human_password_for(
         self._call_log.setdefault(identifier, 0)
         self._call_log[identifier] += 1
         return Fault(
+            'bw secrets human_password {} {} {} {} {}'.format(
```

Comment:
A bit ugly. One might think about automatically adding kwargs to the fault_identifier. I haven’t done that because, well, maybe one of those kwargs is a Fault? Who knows.

Reply:
Don't get it. `digits` might be a fault now, so what?